Competition2017/src/competition/subsystems/fuel_launcher/commands/MaintainShooterPowerCommand.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.fuel_launcher.commands;
+
+import competition.subsystems.shooter.SideShooterSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class MaintainShooterPowerCommand extends BaseCommand {
+
+    final SideShooterSubsystem sideShooter;
+
+    public MaintainShooterPowerCommand(SideShooterSubsystem sideShooter) {
+        this.sideShooter = sideShooter;
+        this.requires(this.sideShooter);
+    }
+
+    @Override
+    public void initialize() {
+    
+    }
+    
+    @Override
+    public void execute() {
+        sideShooter.updateTelemetry();
```

Comment:
The reason that it seems wrong is that it specifically calls out "maintain[ing] power" yet it just allows the motor to keep doing what it's doing, which generally _isn't_ maintaining the power. How about `ContinueShooterActionCommand` or `RunCurrentShooterActionCommand`?

It seems like this functionality might be better-served if we had a command specifically for maintaining speed which explicitly kept the talon performing that action. That feels like a much more natural design to me, as otherwise you're essentially losing direct command of your motors (there isn't any state on our end controlling what it does).